src/api/app/assets/stylesheets/webui/diff.scss

```diff
@@ -14,6 +14,10 @@
   display: none;
 }
 
+.diff-accordion {
+  --bs-accordion-active-bg: var(--bs-secondary-bg);
```

Comment:
We already modigy `--bs-accordion-active-bg` here:

https://github.com/openSUSE/open-build-service/blob/46e11a703d701b52dd17f1697fd358d3a589c9e9/src/api/app/assets/stylesheets/webui/build-results.scss#L180

Should we use a more general solution for both cases?